test-files/old-tests/filters/blacklist_0.cpp

```diff
@@ -0,0 +1,88 @@
+namespace N0
```

Comment:
This is good as an extra but doesn't really scale well when we have unit tests properly set up, which would tell us exactly what broke and where.